### DISCUSSION TITLE
WSPBiexTransform and Gate Vertices

### DISCUSSION THREAD

**Discussion by brianbourke75** [whitews/FlowKit#233] [Q&A] [answered]
I am looking at some gated data and the vertices listed in the min/max values of the Dimensions for the RectangleGate are not in the same scale as the input data (the rectangle values are a couple of orders of magnitude higher).  I also happened to notice that this data set uses WSPBiexTransform for the dimension in question.  I tried to apply the transform to the rectangle range but without any luck.  Any suggestions on how to handle this?  I have not run into this problem yet with PolygonGates but do I also have to worry about that?

**Comment by whitews** [comment, marked as answer]
Hi Brian,

Gate boundaries are specified in the space defined by the corresponding Dimension's transform, i.e. they are in the transformed space. If you transform the Sample's input values (and apply any needed compensation), the coordinates should match.

Regards,
Scott
